codespell_lib/data/dictionary.txt

```diff
@@ -3809,38 +3898,54 @@ excell->excel
 excellance->excellence
 excellant->excellent
 excells->excels
+excepetion->exception
+excepions->exceptions
 exceptation->expectation
+exceptionss->exceptions
 excercise->exercise
 excercised->exercised
 excerciser->exerciser
 excercises->exercises
 excercising->exercising
 excetion->exception
+excetpion->exceptionts
```

Comment:
```
excetpion->exceptions
exceptionts->exceptions
```

Reply:
fixed, added a few more entries in a363f2b + sorted